pkg/gql/query/blocks_test.go

```diff
@@ -34,14 +38,18 @@ func TestBlocksByHeight(t *testing.T) {
 			"genesis":[
 			  {  
 				"header":{  
-				  "height":0
+				  "height":0,
```

Comment:
To keep the README up to date, could you update section `* Fetch block by hash with full set of supported fields`  with the new ones?